[OSDOCS-9186]: Include a note on DDOS protection Service for OSD on Google Cloud in OSD service definition doc (#69717)
openshift/openshift-docs: modules/sdpolicy-security.adoc

[id="network-security_{context}"]
== Network security

> Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules (AWS Security Groups or Google Cloud Compute Engine firewall rules). {product-title} customers on AWS are also protected against DDoS attacks with link:https://docs.aws.amazon.com/waf/latest/developerguide/ddos-overview.html[AWS Shield Standard].

Reviewer: Maybe rephrase to avoid parenthesis. 
```suggestion
Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules, either AWS Security Groups or Google Cloud Compute Engine firewall rules.
```
